lib/max_subarray.rb

```diff
@@ -1,8 +1,18 @@
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: o(n)
+# Space Complexity: o(1)
 def max_sub_array(nums)
```

Comment:
Your indentation is a little funky here.  Makes it hard to read.  Otherwise it looks good 💯